railties/test/isolation/assets/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "dummy",
+  "private": true,
+  "dependencies": {
+    "@rails/ujs": "file:../../../../actionview",
+    "@rails/webpacker": "^4.0.0-rc.7",
```

Comment:
Could/should this point at the git repo instead?

Reply:
:+1: Totally agree. I fixed.